drivers/painter/oled_panel/qp_oled_panel.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Nick Brassel (@tzarc)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#include "color.h"
+#include "qp_internal.h"
+#include "qp_surface_internal.h"
+
+#ifdef QUANTUM_PAINTER_SPI_ENABLE
+#    include "qp_comms_spi.h"
+#endif // QUANTUM_PAINTER_SPI_ENABLE
+
+#ifdef QUANTUM_PAINTER_I2C_ENABLE
+#    include "qp_comms_i2c.h"
+#endif // QUANTUM_PAINTER_I2C_ENABLE
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Common OLED panel implementation
+
+// Driver vtable with extras
+struct oled_panel_painter_driver_vtable_t {
+    struct painter_driver_vtable_t base; // must be first, so it can be cast to/from the painter_driver_vtable_t* type
+
+    // Opcodes for normal display operation
+    struct {
+        uint8_t display_on;
+        uint8_t display_off;
+    } opcodes;
+};
```

Comment:
```suggestion
typedef struct oled_panel_painter_driver_vtable_t {
    painter_driver_vtable_t base; // must be first, so it can be cast to/from the painter_driver_vtable_t* type

    // Opcodes for normal display operation
    struct {
        uint8_t display_on;
        uint8_t display_off;
    } opcodes;
} oled_panel_painter_driver_vtable_t ;
```